ast/environment.go

```diff
@@ -207,32 +206,7 @@ func (d *EnvironmentDecl) recordSyntax() *syntax.Node {
 	return &d.syntax
 }
 
-// NewDiagnosticWriter returns a new hcl.DiagnosticWriter that can be used to print diagnostics associated with the
-// environment.
-func (d *EnvironmentDecl) NewDiagnosticWriter(w io.Writer, width uint, color bool) hcl.DiagnosticWriter {
-	fileMap := map[string]*hcl.File{}
-	if d.source != nil {
-		if s := d.syntax; s != nil {
-			fileMap[s.Syntax().Range().Filename] = &hcl.File{Bytes: d.source}
-		}
-	}
-	return newDiagnosticWriter(w, fileMap, width, color)
-}
-
-func EnvironmentSyntax(node *syntax.ObjectNode, description *StringExpr, imports ImportListDecl, values PropertyMapDecl) *EnvironmentDecl {
-	return &EnvironmentDecl{
-		syntax:      node,
-		Description: description,
-		Imports:     imports,
-		Values:      values,
-	}
-}
-
-func Environment(description *StringExpr, imports ImportListDecl, values PropertyMapDecl) *EnvironmentDecl {
-	return EnvironmentSyntax(nil, description, imports, values)
-}
```

Comment:
Appears unused. Checked both `pulumi` and `pulumi-service`. But please correct me if I'm wrong!